bsip-0086.md

```diff
@@ -0,0 +1,115 @@
+    BSIP: 0086
+    Title: Share market fees to the network
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-11-29
+    Discussion: https://github.com/bitshares/bsips/issues/194
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change: when every trade happens, if there are
+market fees generated, a portion (E.G. 5% or 10%) of the market fees goes to
+the network.
+
+# Motivation
+
+The network needs more tools to generate income to support its development.
+
+# Rationale
+
+A major activity in the network is trading assets. The asset owners make
+profits in the form of market trading fees. Currently, the main tools for the
+network to generate income from asset trading activities are order creation
+fee and order cancellation fee. It seems reasonable that the asset owners
+share some profits (a part of market fees) to the network to support its
+development.
+
+The cut of market fees can go to committee-account's vesting balances.
+As a supporting measure, committee-account should be exempted from
+white-listing restrictions, so that it is able to claim the vesting balances
+and sell them for core token or other tokens to pay for development later.
+
+# Specifications
+
+## Protocol upgrade
+
+A time will need to be scheduled for applying the change. In this document,
+terms "before the protocol upgrade", "at the protocol upgrade" and "after
+the protocol upgrade" may or may not be used to indicate things happen before
+the scheduled time, at the scheduled time and after the scheduled time.
+
+## New global parameter
+
+Add a new global parameter `market_fee_network_percent` which can be updated
+by the committee only after the protocol upgrade.
+Initial value of that parameter is `0`.
+Valid range of that parameter is `[0, 100%]`.
```

Comment:
Here is a thought: Does it make sense to couple this into strengthening the referral rewards program.

What I mean: Whatever comes in as `market_fee_network_percent` gets funneled into the referral rewards program, and is split according to the `network_fee` (with current values that means that 20% goes to committee, and 80% into referral rewards.

Downside is that this also potentially clutters account's balances with useless assets, but it would give a massive boost to the rewards program.

Reply:
Another side effect is the referrers may get tokens they can't use due to white-lists and etc, it could also be a headache for asset owners. Although the idea is interesting, I personally prefer not adding it to this BSIP (maybe create a new BSIP).

Reply:
Allow asset onwers to turn off rewards funneling? (then all goes to committee)?

Reply:
Sounds complicated.

Reply:
Ok, I guess if asset owners what to take part in rewards program to boost their adoption they can set it themselves.

Reply:
Just one last I want to add: A strong referral rewards program is exactly what other exchanges have that makes them grow massively. So we are putting the success in the hands of the gateways. Just FYI.